Pyjion/absint.cpp

```diff
@@ -1887,6 +1933,22 @@ JittedCode* AbstractInterpreter::compile_worker() {
             vector<bool>()
         )
     );
+
+    for (size_t i = 0; i < m_code->co_argcount + m_code->co_kwonlyargcount; i++) {
+        auto local = get_local_info(0, i);
+        if (!local.ValueInfo.needs_boxing()) {
+            m_comp->emit_load_fast(i);
+
+            if (local.ValueInfo.Value->kind() == AVK_Float) {
+                m_comp->emit_unbox_float();
+                m_comp->emit_store_local(get_optimized_local(i, AVK_Float));
+            }
+            else if (local.ValueInfo.Value->kind() == AVK_Integer) {
+                m_comp->emit_unbox_int_tagged();
+                m_comp->emit_store_local(get_optimized_local(i, AVK_Any));
```

Comment:
Nope, AVK_Any corresponds to a generic pointer type which is what we want for our integers because they're always tagged pointers.  It actually wouldn't make any difference if I did because AVK_Float is the only type that we give a different type for currently.